hw/ip/otbn/data/otbn.hjson

```diff
@@ -134,17 +134,34 @@
   registers: [
     { name: "CMD"
       desc: "command register",
-      swaccess: "r0w1c",
+      swaccess: "wo",
       hwaccess: "hro",
       hwext: "true",
       hwqe:  "true",
       fields: [
-        { bits: "0"
-          name: "start"
+        { bits: "1:0"
+          name: "cmd"
+          resval: 0,
           desc: '''
-            Start the operation
+            Initiates an OTBN operation.
+
+            1 (EXECUTE): Start the execution of the program stored in the
```

Comment:
@cdgori can you check if this requires multi bit encoding?

Reply:
I opened https://github.com/lowRISC/opentitan/issues/7822 to track this and unblock this PR. I reserved 8 bits for the CMD register and made it an enum to make it easy to switch to a multi-bit encoding.

Reply:
Thanks